book/src/Usage.md

```diff
@@ -58,3 +58,53 @@ spec:
 ```
 
 [csi-volume]: https://kubernetes.io/docs/concepts/storage/volumes/#csi
+
+
+#### CSI Provisioner Parameters
+
+The CSI dynamic provisioner makes `CreateVolumeRequest` and `DeleteVolumeRequest` calls to CSI drivers.
+The `controllerCreateSecrets` and `controllerDeleteSecrets` fields in those requests can be populated 
+with data from a Kubernetes `Secret` object by setting `csiProvisionerSecretName` and `csiProvisionerSecretNamespace`
+parameters in the `StorageClass`. For example:
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fast-storage
+provisioner: com.example.team/csi-driver
+parameters:
+  type: pd-ssd
+  csiProvisionerSecretName: fast-storage-provision-key
+  csiProvisionerSecretNamespace: pd-ssd-credentials
+```
+
+The `csiProvisionerSecretName` and `csiProvisionerSecretNamespace` parameters
+may specify literal values, or a template containing the following variables:
+* `${pv.name}` - replaced with the name of the PersistentVolume object being provisioned
+
+Once the CSI volume is created, a corresponding Kubernetes `PersistentVolume` object is created.
+The `controllerPublishSecretRef`, `nodeStageSecretRef`, and `nodePublishSecretRef` fields in the 
+`PersistentVolume` object can be populated via the following storage class parameters:
+
+* `controllerPublishSecretRef` in the PersistentVolume is populated by setting these StorageClass parameters:
+  * `csiControllerPublishSecretName`
+  * `csiControllerPublishSecretNamespace`
+* `nodeStageSecretRef` in the PersistentVolume is populated by setting these StorageClass parameters:
+  * `csiNodeStageSecretName`
+  * `csiNodeStageSecretNamespace`
+* `nodePublishSecretRef` in the PersistentVolume is populated by setting these StorageClass parameters:
+  * `csiNodePublishSecretName`
+  * `csiNodePublishSecretNamespace`
+
```

Comment:
Looks great, the only think I would suggest is to add an example with variables mentioned below, I think it will help understand better their usage.

Reply:
added an example